trimmer_decorator.rb

```diff
@@ -0,0 +1,9 @@
+require './base_decorator'
+
+class TrimmerDecorator < BaseDec
+  def correct_name
+    return @nameable.correct_name[0..10] if @nameable.correct_name.length <= 10
```

Comment:
- [ ] kindly know that it should return a maximum of 10 characters, therefore kindly change `[0..10] ` to `[0..9]` :+1: